tests/package.lisp

```diff
@@ -32,7 +32,10 @@
    (#:slice #:coalton-library/slice)
    (#:hashtable #:coalton-library/hashtable)
    (#:iter #:coalton-library/iterator)
-   (#:list #:coalton-library/list)))
+   (#:list #:coalton-library/list)
+   (#:red-black/tree #:coalton-library/red-black/tree)
```

Comment:
Is it important for users of Coalton that these use a red-black tree under the hood? Will we ever want multiple persistent map implementations? Or should these be called tree and map, and we can change the underlying implementation later if desired?

Reply:
I could imagine wanting both red-black trees (Ord trees) and finger trees (prefix trees) in the standard library. But I don't think that the specific red-black implementation is necessary. I'm open to different names for these; in addition to just naming them `tree:Tree` and `map:Map`, I'd pitch `ord-tree:Tree` and `ord-map:Map`.

Reply:
I'm fine with either

Reply:
renamed the files to `ord-tree` and `ord-map`; lmk if you have any objections.